### ISSUE TITLE
valid-template-root should allow multiple tags if part of same v-if

### ISSUE THREAD

**Issue by rightaway** [vuejs/eslint-plugin-vue#884]
`valid-template-root` gives an error if you have the following but it is allowed by Vue so this shouldn't give an error. Same applies if you have 1 or more v-else-if elements there.

```html
<template>
  <div v-if=condition>if</div>
  <div v-else>else</div>
</template>

**Comment by mysticatea** [comment]
Thank you for this issue, but I could not reproduce it: [online demo](https://mysticatea.github.io/vue-eslint-demo/#eJxNjkELwjAMhf/KyHlh4HHU/Qk9eilrKoGalLYr6Nh/t5soXgJ5X/LeW2FWRzCCKfSIwRaabtJ1xnHtKrI/zyqOC6tM7M3Q5H9OIdO0jy8xw88GekhLoAzjCnWhQRSbl18yyx0rek24B8B46g9ebWCH33dMqqWxrQcWR1Iu/KLj9rNen3FvnaOdqSVFmzKlJlCOiQi2N4pSSl4=)

Please use the latest version.

**Comment by jbmoelker** [comment]
@andre-brdoch I experienced the same issue. However when I removed `eslint-plugin-vue-a11y` from my setup, `eslint-plugin-vue` was no longer throwing the `valid-template-root` error. 🤷‍♂
So seems to be an issue of either `eslint-plugin-vue-a11y` or the combination of the two.

**Comment by rightaway** [comment]
@andre-brdoch What do you think the issue is if it works in the demo but not in your editor?
